providers/google/src/airflow/providers/google/cloud/transfers/bigquery_to_gcs.py

```diff
@@ -275,6 +275,9 @@ def execute(self, context: Context):
         else:
             job.result(timeout=self.result_timeout, retry=self.result_retry)
 
+            # issue-59344: Returning destination URIs to be persisted in XCom
```

Comment:
Could you please remove both comments (here and in `execute_complete`), and instead add a docstring in the class level?

You may take inspiration from [DbtCloudRunJobOperator](https://github.com/apache/airflow/blob/8256ee16d1de5b47ad996af82608d2bb836dfe1d/providers/dbt/cloud/src/airflow/providers/dbt/cloud/operators/dbt.py#L87)'s docstring.

I think that it would be more informative for the end-user.